modules/graceful-restart.adoc

```diff
@@ -180,4 +180,4 @@ ip-10-0-211-16.ec2.internal    Ready    control-plane,master   82m   v1.33.4
 ip-10-0-250-100.ec2.internal   Ready    worker                 69m   v1.33.4
 ----
 +
-If the cluster did not start properly, you might need to restore your cluster using an etcd backup.
+If the cluster did not start properly, you might need to restore your cluster using an etcd backup. For more information, see "Restoring to a previous cluster state".
```

Comment:
Note that the text around the assembly's xref will be removed by the main RelatedLinks PR